src/components/slide_in_panel.tsx

```diff
@@ -24,6 +24,8 @@ export interface SlideInPanelProps {
 
 export class SlideInPanel extends React.Component<SlideInPanelProps, {}> {
 
+  public static displayName = 'SlideInPanel';
```

Comment:
For classes it's not needed as they are already named by their class.

Reply:
Ahhhhhh, I'll remove them